blueocean-dashboard/src/main/js/creation/credentials/CredentialsApi.js

```diff
@@ -0,0 +1,102 @@
+import { capabilityAugmenter, Fetch, UrlConfig } from '@jenkins-cd/blueocean-core-js';
+import TempUtils from '../TempUtils';
+
+export class CredentialsApi {
```

Comment:
Credentials arent specifc to creation. Ie slaves use credentials api. Should we be putting this in core js instead?